src/cargo/util/config.rs

```diff
@@ -541,6 +630,7 @@ impl Config {
         !self.frozen && !self.locked
     }
 
+    // TODO: this was pub for RLS but may not be needed anymore?
```

Comment:
If it's not too hard I think this'd be good to stick around, I believe `cargo-vendor` also makes use of this